DTAdmin/MainTabController.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ViewController.swift
+//  DTAdmin
+//
+//  Created by ITA student on 10/10/17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import UIKit
+
+class MainTabController: UITabBarController, UITabBarControllerDelegate {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.delegate = self
+        
+        // Create Tab one
+        let tabOne = UINavigationController()
+        tabOne.setViewControllers([TabOneViewController()], animated: true)
+        let tabOneBarItem = UITabBarItem(title: "Tab 1", image: nil, selectedImage: nil)
+        tabOne.tabBarItem = tabOneBarItem
+        
+        // Create Tab two
+        let tabTwo = TabTwoViewController()
+        let tabTwoBarItem2 = UITabBarItem(title: "Tab 2", image: nil, selectedImage: nil)
+        tabTwo.tabBarItem = tabTwoBarItem2
+        
+        self.viewControllers = [tabOne, tabTwo]
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+    }
+    
+    // UITabBarControllerDelegate method
+    func tabBarController(_ tabBarController: UITabBarController, didSelect viewController: UIViewController) {
+        print("Selected \(viewController.title!)")
```

Comment:
Avoid using force unwraps. Please change to if let or ?? (e.g. viewController.title ?? "" at least)